easybuild/easyconfigs/w/Wannier90/Wannier90-1.2-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'Wannier90'
+version = '1.2'
+
+homepage = 'http://www.wannier.org'
+description = """A tool for obtaining maximally-localised Wannier functions"""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.wannier.org/code']
+
+# ugly hack
+prebuildopts = 'rm make.sys; touch make.sys; '
+prebuildopts += 'F90=ifort FCOPTS="$FFLAGS" LDOPTS="$FFLAGS" LIBDIR="$LAPACK_LIB_DIR" LIBS="$LIBLAPACK" '
```

Comment:
@migueldiascosta please use `$F90` rather than hardcoding `ifort`, that will make the use of `--try-toolchain` works as expected: `F90="$F90"`

This may look a bit silly, but it makes sense if the `Makefile` (or `make.sys`) has a hardcoded value for `F90` in place; defining an environment variable `$F90` (which is done by the toolchain mechanism in the EasyBuild framework) and specifying `F90=` on the `make` command line are not equivalent...